TESTS/mbedmicro-rtos-mbed/basic/main.cpp

```diff
@@ -28,7 +32,7 @@
 #endif
 
 //FastModels not support time drifting test
-#if defined(__ARM_FM)
+#if defined(TARGET_ARM_FM)
```

Comment:
This can be removed, as we already got `#if defined(SKIP_TIME_DRIFT_TESTS)`

Reply:
fixed.